scripts/generate-google3-docs/resource-reference/templates/vertexai_vertexaidataset.tmpl

```diff
@@ -0,0 +1,54 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+{{template "alphadisclaimer.tmpl" .}}
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>Vertex AI</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/vertex-ai/docs">/vertex-ai/docs</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>v1/projects.locations.datasets</td>
```

Comment:
Nit: We probably want to use v1beta1 as KCC probably talks to their beta endpoint (it depends on the configuration in the TF lib).

Reply:
I think you are right but how do I confirm which API endpoint it is using?

Reply:
You could write a mock :-)

Reply:
Or you can just record the http traffic using something like:

```
WRITE_GOLDEN_OUTPUT=1 ARTIFACTS=$(pwd)/artifactz/gcp/ \
E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=real \
   go test -test.count=1 -timeout 3600s -v ./tests/e2e -run "TestAllInSeries/${FIXTURE}" 2>&1 | tee log
```

Reply:
Thank you! Updated to use `v1beta1` in docs.

Reply:
> I think you are right but how do I confirm which API endpoint it is using?

It can also be found in code: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/transport/config.go#L565